CDCgov/NEDSS-Modernization
# [CNFT1-1557] Inactive demographics

## Description
The Patient Demographics finders were only looking for ACTIVE Patients.  The demographics should appear on the Patient Profile regardless of it's status.

## Tickets

* [CNFT1-1557](https://cdc-nbs.atlassian.net/browse/CNFT1-1557)

## Steps to verify

1. Create a new Patient with at least a name
2. View the Patient Profile
3. Delete the Patient
4. Search for the deleted Patient.  (Patient ID search is easiest)
5. View the Patient Profile and ensure that the Name is visible in the Demographics tab


[CNFT1-1557]: https://cdc-nbs.atlassian.net/browse/CNFT1-1557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
